cmd/termui/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -182,16 +182,23 @@ func (wr *WidgetsRender) prepareChainInfo(numMillisecondsRefreshTime int) {
 	synchronizedRound := wr.presenter.GetSynchronizedRound()
 	currentRound := wr.presenter.GetCurrentRound()
 
-	var syncingStr, remainingTimeMessage, blocksPerSecondMessage string
+	nodesProcessed := wr.presenter.GetAccountsSyncNodesProcessed()
+	isNodeSyncingAccounts := nodesProcessed != 0
+
+	var syncingStr, statusMessage, blocksPerSecondMessage string
 	switch {
 	case synchronizedRound < currentRound:
 		syncingStr = statusSyncing
 
 		remainingTime := wr.presenter.CalculateTimeToSynchronize(numMillisecondsRefreshTime)
-		remainingTimeMessage = fmt.Sprintf("Synchronization time remaining: ~%s", remainingTime)
+		statusMessage = fmt.Sprintf("Synchronization time remaining: ~%s", remainingTime)
 
 		blocksPerSecond := wr.presenter.CalculateSynchronizationSpeed(numMillisecondsRefreshTime)
 		blocksPerSecondMessage = fmt.Sprintf("%d blocks/sec", blocksPerSecond)
+	case isNodeSyncingAccounts:
```

Comment:
if we'll update `synchronizedRound` and `currentRound` earlier in the future, this feature won't work since it will go into the first `case` of the `switch`. can remain as it is for now

Reply:
Also: what happens when the node is synced? The number of nodes synced (I don't think) will get a reset, so the termui will always display the trie sync

Reply:
moved `case isNodeSyncingAccounts` on top

as for the second question, the metrics are update right after [reset](https://github.com/ElrondNetwork/elrond-go/blob/termui_trie_sync_status/state/syncer/baseAccountsSyncer.go#L161), when the sync is done